app/src/main/java/com/tvtracker/MainActivity.kt

```diff
@@ -466,7 +466,7 @@ class MainActivity : ComponentActivity() {
     @Composable
     fun UserMediaItemColumn(mediaItems: List<MediaItem>) {
         LazyColumn() {
-            itemsIndexed(mediaItems) { index, item ->
+            itemsIndexed(mediaItems) { _, item ->
```

Comment:
Renamed "index" to "-" since it was never used

Reply:
I'd keep it as it were.  _ might be confusing.